README.md

```diff
@@ -32,6 +32,28 @@ For more information, please refer to:
 - [Getting Started](https://create-react-app.dev/docs/getting-started) – How to create a new app.
 - [User Guide](https://create-react-app.dev) – How to develop apps bootstrapped with Create React App.
 
+## Template structure
+
+```
+.
+├── template
+│   ├── public
+│   │   ├── ...
+│   ├── src
+│   |   └── ...
+│   ├── gitignore
+│   └── README.md
+├── .gitignore
+├── LICENSE
+├── package.json
+├── README.md
+└── template.json
+```
+
+We use `Typescript` by default for our React applications. Along with the standard files from a `create-react-app` 
+project, the folder structure in the `src` folder is created as per our project structure
```

Comment:
```suggestion
project, the folder structure in the `src` folder is created as per the
[React Conevention](https://nimblehq.co/compass/development/code-conventions/react/#project-structure).

```

Reply:
@noor-ahmed fixed in [d180505](https://github.com/nimblehq/react-templates/pull/14/commits/d1805051c58a4efaf844a97558566088ffb20734)